include/swift/AST/DiagnosticsParse.def

```diff
@@ -1030,6 +1030,8 @@ ERROR(expected_await_not_async,none,
 // Yield Statment
 ERROR(expected_expr_yield,PointsToFirstBadToken,
       "expected expression in 'yield' statement", ())
+ERROR(unexpected_label_yield,none,
+      "unexpected label in yield", ())
```

Comment:
Hey @hamishknight, I wonder if it would make sense to have it like `'yield'` or even like the above `in 'yield' statement` maybe?

Reply:
Good catch! Yeah definitely best to be consistent here